docker-for-aws/index.md

```diff
@@ -49,7 +51,9 @@ using CloudFormation. For more about stable and edge channels, see the
 
 
 #### Test Channel
-<p>This is the test channel. It is used for testing release candidates for upcoming releases. Use this channel if you want to test out the new releases before they are available.</p>
+<p>This is the test channel. It is used for testing release candidates for
```

Comment:
why the html tags?

Reply:
Yeah, it's probably better to just make newlines instead of the `<p>` tags.